### PR TITLE
Add readme

### PR DESCRIPTION
**Pull request by redferret**
## Pull request checklist
Please check if your PR fulfills the following requirements:
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Rubocop has passed locally and any fixes were made for failures
- [ ] Build is passing
## Pull request type
Please check the type of change your PR introduces:
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):
## What is the current behavior?

Issue Number: 
closes #
## What is the new behavior?

- 
- 
- 
## Does this introduce a breaking change?
- [ ] Yes
- [ ] No

### CODE REVIEW

**Review by twarbelow — APPROVED** [review]
thank you for putting this together!

File: README.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+## Drink This
+### Live at
+
+https://https://drink-this-frontend.herokuapp.com/
+
+## About this Project
+
+Describe the project
```
Comment (markcyen) [inline README.md:8]: Maybe we should put something like...

"Drink This is a cocktail recommendation app that provides cocktail recommendations to the user. In the front end of this app, we used ReactJS to provide a clean feel. Movement between pages is smooth and seamless, as opposed to rendering views that may take time to load. Another look and feel to this is the frontend interaction with the backend. We displayed simple thumbnails of the cocktails consumed from the CocktailDB API from the backend."

File: README.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+- [Google Auth](https://developers.google.com/identity/protocols/oauth2)
+
+## Testing
+Tests with [Jest](https://jestjs.io/docs/getting-started)
+
+## To Add changes
+
+- Create PR
+- Add the PR to the Viewing Party Project
```
Comment (markcyen) [inline README.md:36]: You mean Drink This Frontend project?

**Review by markcyen — APPROVED** [review]
Cool!

**Review by zachjamesgreen — APPROVED** [review]

**Pull request merged**